### ISSUE TITLE
Shorthand names showing up in web view

### ISSUE THREAD

**Issue by shaddygarg** [zdresearch/OWASP-Nettacker#103]
Both vuln and vulnerability are showing up in web view. Same is the case with info.
![image](https://user-images.githubusercontent.com/21990986/38456727-9210f296-3aa5-11e8-8862-197f315b27b2.png)
_________________
**OS**: `Ubuntu`

**OS Version**: `16.04`

**Web Browser**: `Firefox`

**Comment by Ali-Razmjoo** [comment]
Hey Shaddy,

thanks for creating the issue, I think we may create a new `dict` for synonyms, in that case, we can locate it in the `core/config.py`.

```python
def _profiles():
    """
    a shortcut and users profile to run customize scans (could be modify by user)
    Returns:
        a JSON contains profile names and module names for each
    """
    return {
        "information_gathering": ["port_scan"],
        "vulnerability": ["*_vuln"],
        "scan": ["*_scan"],
        "brute": ["*_brute"]
    }

def _synonym_profile():
    return {
        "info": "information_gathering",
        "vuln": "vulnerability",
    }
```

we should also modify `core/args_load.py` and `api/api_core.py->__rules` to apply synonyms and replace them. in this way, synonyms will not be shown in web or API. let me know if you would like work on this, and if it's little complicated, I can take care of it.

Best Regards.

**Comment by shaddygarg** [comment]
Hey Ali,
I would love to work on this issue. Along with this, I think we should also make change in `api/api_core.py->__scan_methods` to prevent it from showing in HTML view. I am however unsure about the change in core/args_load.py. Are we looking to exclude these in the arguments too?
Regards.
Regards.

**Comment by Ali-Razmjoo** [comment]
in that case, you can remove the synonyms from there, but if we have to define synonyms first!

**Comment by shaddygarg** [comment]
The synonyms can be defined in the `core/config.py` file and the `api/api_core.py`can import them to exclude from the profiles shown in the API and the web view. I have implemented the same in #104.

**Comment by shaddygarg** [comment]
Closed via #104
